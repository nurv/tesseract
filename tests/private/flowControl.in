int i=5;
for(int j=0; j<i; j++){
	System.out.println("ho ho");
}
if(i==4){
	i++;
}else{
i--;
}
int j=0;
while(j<i){
	System.out.println("j=" + j++);
}
switch(j){
	case 4:
			System.out.println("j=" + j);
	default:
			System.out.println("bling");
}
