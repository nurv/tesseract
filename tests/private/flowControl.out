> > ho ho
ho ho
ho ho
ho ho
ho ho
> > > j=0
j=1
j=2
j=3
> j=4
bling
> 